client/src/components/HomeCss.css

```diff
@@ -0,0 +1,26 @@
+header{
```

Comment:
File name is a bit redudant. I think just `home.css` would be fine rather than repeating css.